go.mod

```diff
@@ -1,15 +1,13 @@
 module github.com/grafana/crossplane-provider-grafana
 
-go 1.24.6
-
-toolchain go1.24.7
+go 1.24.7
 
 require (
 	dario.cat/mergo v1.0.2
 	github.com/alecthomas/kingpin/v2 v2.4.0
-	github.com/crossplane/crossplane-runtime v1.20.0
-	github.com/crossplane/crossplane-tools v0.0.0-20250603090330-889cfb100517
-	github.com/crossplane/upjet v1.9.0
+	github.com/crossplane/crossplane-runtime/v2 v2.1.0
```

Comment:
Let’s update to v2.0 first. That’s the recommended approach because Crossplane performs CRD migrations at each of those versions.

Reply:
On a similar note, see https://github.com/grafana/deployment_tools/issues/392040#issuecomment-3546652776. Keeping this comment separate since this is about deployment tools.